MonoXAnalysis/python/samples/samples_13TeV_VJETS_RunIISpring16MiniAODv2.py

```diff
@@ -0,0 +1,30 @@
+import PhysicsTools.HeppyCore.framework.config as cfg
+import os
+
+from CMGTools.RootTools.samples.ComponentCreator import ComponentCreator
+kreator = ComponentCreator()
+
+
```

Comment:
these BKG samples should be added in 
/RootTools/python/samples/samples_13TeV_RunIISpring16MiniAODv2.py